pkg/service/service.go

```diff
@@ -108,7 +108,7 @@ func List(client *occlient.Client, applicationName string, projectName string) (
 	return services, nil
 }
 
-// ListWithSmartStatus lists all the deployed services and additionally resolve provide the "smart" status of each one
+// ListWithSmartStatus lists all the deployed services and additionally provides a "smart" status for each one of them
```

Comment:
What means `smart` ? That should be interesting to add some values between parenthesis

Reply:
ListWithDetailedStatus any better?